apps/molgenis-components/src/components/account/Interfaces.ts

```diff
@@ -2,9 +2,11 @@ import { ISetting } from "../../Interfaces/ISetting";
 
 export interface ISession {
   email?: string;
+  locale?: string;
   roles?: string[];
   schemas?: any;
   settings?: ISetting[];
+  settingsMap?: any;
```

Comment:
This is not being used, so it should not be on here (the settingsMap is a local variable in the vue file).

The typing of a map is mostly a Record, here it would be 'Record<string, string>'

Reply:
ah yes, nice